app/jobs/validation_retry_job.rb

```diff
@@ -6,46 +6,12 @@ class ValidationRetryJob < CronJob
   queue_as :validation_retry
 
   def perform
-    ECFParticipantValidationData.where(api_failure: true).each do |validation_data|
-      validation_result = ParticipantValidationService.validate(trn: validation_data.trn,
-                                                                full_name: validation_data.full_name,
-                                                                date_of_birth: validation_data.date_of_birth,
-                                                                nino: validation_data.nino)
-      ActiveRecord::Base.transaction do
-        validation_data.update!(api_failure: false)
-        if validation_result.present?
-          eligibility_data = store_eligibility_data!(validation_data, validation_result)
-          eligibility_data.manual_check_status! unless store_trn!(validation_result[:trn], validation_data.participant_profile)
-
-          validation_data.destroy! if eligibility_data.eligible_status?
-        end
-      end
+    ECFParticipantValidationData.where(api_failure: true).find_each do |validation_data|
+      ValidateParticipant.call(participant_profile: validation_data.participant_profile)
```

Comment:
Does this need to pass first name only config?

Reply:
Good catch!